src/client/dfs/dfs.c

```diff
@@ -4958,7 +4958,13 @@ dfs_access(dfs_t *dfs, dfs_obj_t *parent, const char *name, int mask)
 }
 
 int
-dfs_chmod(dfs_t *dfs, dfs_obj_t *parent, const char *name, mode_t mode)
+dfs_access(dfs_t *dfs, dfs_obj_t *parent, const char *name, int mask)
+{
+	return dfs_access_wflag(dfs, parent, name, mask, 0);
```

Comment:
generally it's not great for an API to call another API. in some cases it's unavoidable, but in this case, it is and it would be better to do something like:

access_int(.., int flags)

dfs_access() {access_int(..., 0);}
dfs_access_wflags() {access_int(..., flags);}

Reply:
Sounds good. I will fix it. Thank you!